src/state.rs

```diff
@@ -40,25 +43,35 @@ impl VMState {
     pub fn new(program_code: Vec<U256>) -> Self {
         Self {
             registers: [U256::zero(); 15],
-            flag_lt: false,
+            flag_lt_of: false,
             flag_gt: false,
             flag_eq: false,
             current_frame: CallFrame::new(program_code),
         }
     }
 
-    pub fn new_with_flag_state(flag_lt: bool, flag_eq: bool, flag_gt: bool) -> Self {
+    pub fn new_with_flag_state(flag_lt_of: bool, flag_eq: bool, flag_gt: bool) -> Self {
         let registers = [U256::zero(); 15];
         let current_frame = CallFrame::new(vec![]);
         Self {
             registers,
-            flag_lt,
+            flag_lt_of,
```

Comment:
Why the rename?

Reply:
I originally thought this flag was only "less than", but upon re-reading the spec, I've noticed it's actually  "overflow or less than", hence the rename.

Reply:
Correct, [here](https://github.com/matter-labs/zksync-era/blob/main/docs/specs/zk_evm/vm_specification/zkSync_era_virtual_machine_primer.md#vm-architecture) it states nothing about overflow, yet it is in the [specification document](https://github.com/matter-labs/zksync-era/blob/main/docs/specs/zk_evm/vm_specification/EraVM_formal_specification.pdf).